pkg/resource/resource.go

```diff
@@ -601,3 +604,33 @@ func checkForImagePullBackOff(obj *unstructured.Unstructured, namespace string)
 
 	return fmt.Errorf("unexpected Phase of Pods in DameonSet: %s", obj.GetName())
 }
+
+func sendNodesMetrics(obj *unstructured.Unstructured, crName string) {
```

Comment:
I would also use `client.Object` instead of `*unstructured.Unstructured` here, but I would wait for unit tests first.

Reply:
same comment as above